KIPs/kip-12.md

```diff
@@ -0,0 +1,67 @@
+---
+kip: 12
+title: KyberDMM Expansion and Liquidity Mining on BSC
+status: Active
+type: Mint KNC
+authors: Kyber Network team
+created: 2021-08-10
+---
+
```

Comment:
We need details on proposal parameters such as:
1. start time
2. end time
3. execution
4. executor

Please take a look at other KIPs as a reference

Reply:
Sure,
For now can I just include these

executor: long timelock executor
voting_power: current staked voting power
execution: transfer KNC to TBD
start_timestamp: TBD
end_timestamp: TBD